src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -112,7 +116,7 @@ private async Task<string> GetBundle(HttpClient httpClient)
             return bundlePath;
         }
 
-        internal bool TryLocateExtensionBundle(out string bundlePath)
+        internal bool TryLocateExtensionBundle(out string bundlePath, FunctionsHostingConfigOptions configOption)
```

Comment:
Do we need this to be passed as a function parameter if it's a global variable in this class?

Reply:
It does not, for some reason i thought it was a static method. Have fixed this